app/lib/actions/katello/repository/sync.rb

```diff
@@ -97,6 +99,14 @@ def pulp_task_id
           end
         end
 
+        def regenerate_applicability(repo)
+          if SETTINGS[:katello][:katello_applicability]
+            plan_action(Actions::Katello::Applicability::Repository::Regenerate, :repo_id => repo.id)
```

Comment:
We probably want to pass contents_changed here, to skip applicability calcs if nothing has changed.  We can tackle that as a 2nd issue, or in this one. Preferences?

Reply:
It sounds like a relatively minor change for the PR, I'll add it in.